packages/votingV2/schema.graphql

```diff
@@ -1,4 +1,4 @@
-type Globals @entity {
+type Global @entity {
```

Comment:
is renaming this a breaking change to the api?

Reply:
No, it's not, this entity wasn't used before.